packages/storey/src/containers/item.rs

```diff
@@ -175,6 +175,24 @@ where
         self.storage.set(&[], &bytes);
         Ok(())
     }
+
+    /// Remove the value of the item.
+    ///
+    /// # Example
+    /// ```
+    /// # use mocks::encoding::TestEncoding;
+    /// # use mocks::backend::TestStorage;
+    /// use storey::containers::Item;
+    ///
+    /// let mut storage = TestStorage::new();
+    /// let item = Item::<u64, TestEncoding>::new(0);
+    ///
```

Comment:
```suggestion
    ///
    /// item.access(&mut storage).set(&42).unwrap();
```

Not a must, but this probably makes a better doc test here.